cmd/opm/alpha/template/cmd.go

```diff
@@ -1,29 +1,27 @@
 package template
 
 import (
-	"io"
-	"os"
-
 	"github.com/spf13/cobra"
 )
 
 func NewCmd() *cobra.Command {
+	var output string
+
 	runCmd := &cobra.Command{
 		Use:   "render-template",
 		Short: "Render a catalog template type",
 		Args:  cobra.NoArgs,
 	}
 
-	runCmd.AddCommand(newBasicTemplateCmd())
-	runCmd.AddCommand(newSemverTemplateCmd())
+	bc := newBasicTemplateCmd()
+	// bc.Hidden = true
```

Comment:
Remove?

Reply:
A planned follow-up in https://github.com/operator-framework/operator-registry/issues/1330 will use these.  I was initially doing it as part of this PR, but realized that it would balloon this PR and likely impact the speed of its review.  If you don't feel strongly, I'd just leave these since for some time the commands would be hidden-but-callable.